app/frontend/src/cljs/teet/meeting/meeting_view.cljs

```diff
@@ -827,7 +827,7 @@
 (defn add-decision-component
   [e! meeting agenda-topic]
   (r/with-let [[pfrom pto] (common/portal)]
-    [:div {:class (<class common/hierarchical-container-style (as-hex (lighten theme-colors/gray-lighter 5)))}
+    [:div {:class (<class common/hierarchical-container-style (as-hex (lighten theme-colors/white 5)))}
```

Comment:
Doe this need the common/hierarchical-container-style class? Since the meeting view now shouldn't use the hierarchical container at all?